avaframe/com1DFAPy/com1DFA.py

```diff
@@ -375,7 +448,7 @@ def placeParticles(mass, indx, indy, csz, massPerPart):
     return xpart, ypart, mPart, nPart
 
 
-def intializeMassEnt(dem):
+def intializeMassEnt(dem, flagEntRes, entLine):
```

Comment:
```suggestion
def initializeMassEnt(dem, flagEntRes, entLine):
```
typo...  -> Maybe even rename it to initializeEntrainmentMass ?